### DISCUSSION TITLE
Running  train.py stand alone offline

### DISCUSSION THREAD

**Discussion by synersignart** [ultralytics/yolov5#5597] [Q&A]
hi guys has any one run Train.py off their own computer oppose to google colaboratory 

I needed something that stands alone for custom train data 

i have download all requirements.txt

every time I run it reports an error message

TensorBoard: Start with 'tensorboard --logdir yolov5\runs\train', view at http://localhost:6006/
2021-11-10 10:57:32.397725: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] 

Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2021-11-10 10:57:32.402098: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

or download 

Yolov5.pt

**Comment by glenn-jocher** [comment]
@synersignart 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃
